controllers/replicationdestination_controller.go

```diff
@@ -288,6 +288,14 @@ func (r *rsyncDestReconciler) ensureJob(l logr.Logger) (bool, error) {
 		}
 		backoffLimit := int32(2)
 		r.job.Spec.BackoffLimit = &backoffLimit
+		if !r.Instance.Spec.Rsync.PauseSync {
```

Comment:
should this be an if...else block? you only save one line of code, not sure it really matters other than readability and style...

Reply:
I swear I have written code before :facepalm: